README.md

```diff
@@ -38,6 +39,15 @@ module "power-infrastructure" {
 }
 ```
 
+You can configure the details in a `input.tfvars` variable definitions file.
+
+```bash
+terraform apply -var-file="input.tfvars"
+```
+
+## Required IAM access policies
+
+No permissions are needed to run this module.
```

Comment:
I don't think that's accurate - probably want to have @surajsbharadwaj inputs on IAM permissions here.

Reply:
already reached out https://github.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/pull/22#discussion_r947631928